coralproject/talk
# [CORL-2349] Use translation keys for relayNetworkRequestError strings

## What does this PR do?

Use translation keys to get error message text for the `RelayNetworkRequestError` defined in `src/core/client/framework/lib/errors/relayNetworkRequestError.ts`.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

Put the following after line 14 in your `customErrorMiddleware`:

```
const msg = getMessage(
    localeBundles,
    "common-error-relayNetworkRequestError-anUnexpectedNetworkError",
    "THIS FAILED AND USED DEFAULT"
  );
  // eslint-disable-next-line no-console
  console.log("LOOK HERE", msg);
```

Import `getMessage` from `i18n` in `customErrorMiddleware` via:

```
import { getMessage } from "../i18n";
```

Have any error throw client side and see that the console shows the `LOOK HERE` translation as translated correctly from `common.ftl` with `An unexpected network error occured, please try again later.` and is not using the text `THIS FAILED AND USED DEFAULT`.
 
## How do we deploy this PR?

N/A
